### PR TITLE
add support of kubecolor binary

### LINKED ISSUE #1072: Add tool "kubecolor"
Great project! Thank you.

It would be cool to add "kubecolor" (https://github.com/kubecolor/kubecolor).

Releases page: https://github.com/kubecolor/kubecolor/releases

* How many downloads does this tool have? Check at: 

15,447 (https://somsubhra.github.io/github-release-stats/?username=kubecolor&repository=kubecolor&page=1&per_page=5)

* If approved, are you willing to submit a pull request for this?

I'd try but can't promise :D.

## Your Environment

* What Kubernetes distribution are you using?

```
Client Version: v1.26.15
Kustomize Version: v4.5.7
Server Version: v1.28.8+k3s1
```

* Operating System and version (e.g. Linux, Windows, MacOS):

```
Linux hostname 6.7.12-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.7.12-1 (2024-04-24) x86_64 GNU/Linux

PRETTY_NAME="Debian GNU/Linux trixie/sid"
```

* What arkade version is this?

```
Version: 0.11.12
Git Commit: d93098490dd3ba78c556c89a502bdf139c29a458
```

### PR DESCRIPTION
**Pull request by nitishkumar71**
## Description

Add support of kubecolor
## Motivation and Context

- [x] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))
Closes #1072 

## How Has This Been Tested?

Executing `./arkage get kubecolor` produced below output
```
Downloading: kubecolor
2024/06/01 19:35:05 Looking up version for kubecolor
2024/06/01 19:35:05 Found: v0.3.3
Downloading: https://github.com/kubecolor/kubecolor/releases/download/v0.3.3/kubecolor_0.3.3_linux_amd64.tar.gz
2.00 MiB / 2.00 MiB [-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------] 100.00%
/tmp/arkade-2340145679/kubecolor_0.3.3_linux_amd64.tar.gz written.
2024/06/01 19:35:10 Extracted: /tmp/arkade-2340145679/kubecolor
2024/06/01 19:35:10 Copying /tmp/arkade-2340145679/kubecolor to /home/ubuntu/.arkade/bin/kubecolor

Wrote: /home/ubuntu/.arkade/bin/kubecolor (4.764MB)

# Add arkade binary directory to your PATH variable
export PATH=$PATH:$HOME/.arkade/bin/

# Test the binary:
/home/ubuntu/.arkade/bin/kubecolor

# Or install with:
sudo mv /home/ubuntu/.arkade/bin/kubecolor /usr/local/bin/

🚀 Speed up GitHub Actions/GitLab CI + reduce costs: https://actuated.dev

```
If updating or adding a new CLI to `arkade get`, run:

```
go build && ./hack/test-tool.sh TOOL_NAME
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [x] I have updated the list of tools in README.md if (required) with `./arkade get --format markdown`
- [ ] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

- [ ] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -914,6 +915,4 @@ There are 56 apps that you can install on your cluster.
 | [waypoint](https://github.com/hashicorp/waypoint)                            | Easy application deployment for Kubernetes and Amazon ECS                                                                                                       |
 | [yq](https://github.com/mikefarah/yq)                                        | Portable command-line YAML processor.                                                                                                                           |
 | [yt-dlp](https://github.com/yt-dlp/yt-dlp)                                   | Fork of youtube-dl with additional features and fixes                                                                                                           |
-There are 154 tools, use `arkade get NAME` to download one.
-
-> Note to contributors, run `arkade get --format markdown` to generate this list
+There are 155 tools, use `arkade get NAME` to download one.
```
Comment (rgee0) [inline README.md:918]: Did we intend to drop the note to contributors?

**Review by rgee0 — COMMENTED** [review]
Just one query in the readme.

Also, the e2e tests are failing on k10multicluster because theyve renames the binary, from:
https://github.com/kastenhq/external-tools/releases/download/7.0.0/k10multicluster_7.0.0_linux_amd64.tar.gz
to:
https://github.com/kastenhq/external-tools/releases/download/7.0.0/k10tools_7.0.0_linux_amd64.tar.gz

Do you want to fix that here too?

File: README.md (hunk 1 above)
Comment (nitishkumar71) [reply README.md:918]: No it was a miss.

**Comment by nitishkumar71** [comment]
> Just one query in the readme.
> 
> Also, the e2e tests are failing on k10multicluster because theyve renames the binary, from: https://github.com/kastenhq/external-tools/releases/download/7.0.0/k10multicluster_7.0.0_linux_amd64.tar.gz to: https://github.com/kastenhq/external-tools/releases/download/7.0.0/k10tools_7.0.0_linux_amd64.tar.gz
> 
> Do you want to fix that here too?

Apologies, I am not sure about there tooling and what purpose these binary serve. In the last [release](https://github.com/kastenhq/external-tools/releases/tag/6.5.14) both [k10tools](https://github.com/kastenhq/external-tools/releases/download/6.5.14/k10tools_6.5.14_linux_amd64.tar.gz) and [k10multicluster](https://github.com/kastenhq/external-tools/releases/download/6.5.14/k10multicluster_6.5.14_linux_amd64.tar.gz) were present. Now they have dropped k10multicluster binary. If we are sure, those binary were supposed to replace each other then I can go ahead and fix it.

**Comment by rgee0** [comment]
> 
> Apologies, I am not sure about there tooling and what purpose these binary serve. In the last [release](https://github.com/kastenhq/external-tools/releases/tag/6.5.14) both [k10tools](https://github.com/kastenhq/external-tools/releases/download/6.5.14/k10tools_6.5.14_linux_amd64.tar.gz) and [k10multicluster](https://github.com/kastenhq/external-tools/releases/download/6.5.14/k10multicluster_6.5.14_linux_amd64.tar.gz) were present. Now they have dropped k10multicluster binary. If we are sure, those binary were supposed to replace each other then I can go ahead and fix it.

I'm no better informed, and I didn't spot the previous release had both.  It looks like v7 release didn't build `k10multicluster` binaries.  It's unfortunate that there isn't a little more detail on the repo.  I'd suggest ignoring my previous message and not 'fixing' without further detail.

**Comment by alexellis** [comment]
These were added for @MichaelCade. We can probably drop them now given the amount of issues they've caused.

**Review by alexellis — APPROVED** [review]
LGTM

**Comment by MichaelCade** [comment]
> Just one query in the readme.
> 
> Also, the e2e tests are failing on k10multicluster because theyve renames the binary, from: https://github.com/kastenhq/external-tools/releases/download/7.0.0/k10multicluster_7.0.0_linux_amd64.tar.gz to: https://github.com/kastenhq/external-tools/releases/download/7.0.0/k10tools_7.0.0_linux_amd64.tar.gz
> 
> Do you want to fix that here too?

These are two completely different tools

**Comment by MichaelCade** [comment]
> These were added for @MichaelCade. We can probably drop them now given the amount of issues they've caused.

Probably remove, I have never once seen or had anyone ask anything about using arkade to deploy and these are only associated to Kasten which had a sponsored entry a few years back so these tools are only relevant to Kasten deployments.

**Pull request merged**